src/versions.js

```diff
@@ -1,45 +1,220 @@
-                    devLibraryVersion: '2.0.0',
-                    designLibraryVersion: '', // TODO: Add Design Library version
-                    designLibraryDoc: '', // TODO: Add Sketch Cloud Link
+                button: {
+                    designArea: 'Buttons',
+                    designItems: [
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/kJJpag',
+                            name: 'Standard Button Styles',
+                            version: '2.0.0',
+                        },
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/oGGwYY',
+                            name: 'Touchscreen Button Styles',
+                            version: '2.0.0',
+                        },
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/R00L1y',
+                            name: 'Button Types',
+                            version: '2.0.1',
+                        },
+                    ],
+                    devVersion: '2.0.0',
+                },
+                checkbox: {
+                    designArea: 'Checkbox',
+                    designItems: [
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/YVVzGM',
+                            name: 'Standard Checkbox',
+                            version: '2.0.0',
+                        },
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/Kee1vn',
+                            name: 'Mini Checkbox',
+                            version: '2.0.0',
+                        },
+                    ],
+                    devVersion: '2.0.0',
+                },
+                durationPicker: { // TODO: Design needs to add documentation.
+                    designArea: '',
+                    designItems: [],
+                    devVersion: '2.0.0',
+                },
+                input: {
+                    designArea: 'Text Inputs',
+                    designItems: [
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/8KKjya',
+                            name: 'Standard Single Line Input',
+                            version: '2.0.0',
+                        },
+                        {
+                            doc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/a/2ooq5J',
+                            name: 'Button & Single Line Input',
+                            version: '2.0.0',
+                        },
+                    ],
+                    devVersion: '2.0.0',
+                },
+                prompt: {
+                    designArea: 'Button Dropdowns',
```

Comment:
Is this `designArea` correct?

Reply:
Yep, it matches designs docs. Maybe eventually they'll move it, but I probably have to prod them a little bit.